CHANGELOG.md

```diff
@@ -1,3 +1,128 @@
+# Kubermatic 2.17
+
+## [v2.17.0](https://github.com/kubermatic/kubermatic)
+
+### Misc
```

Comment:
We should add a list of supported Kubernetes versions above like we did for other minor releases.

Reply:
from where to get the list?

Reply:
https://github.com/kubermatic/kubermatic/blob/release/v2.17/pkg/controller/operator/common/defaults.go#L193

Reply:
Added